docs/pages/access-controls/sso/okta.mdx

```diff
@@ -137,37 +163,39 @@ spec:
 ```
 
 - Devs are only allowed to login to nodes labelled with `access: relaxed` label.
-- Developers can log in as `ubuntu` user
-- Developers also do not have any "allow rules" i.e. they will not be able to
+- Developers can log in as `ubuntu` user or a user matching their email prefix.
```

Comment:
Is there a way to avoid repeating "Devs"/"Developers" in this list?

Reply:
I reformatted it a bit, but it could still be considered redundant. Happy to take any suggestions to further improve this section.